include/config/save.h

```diff
@@ -20,4 +20,6 @@
 
                                             // Grand Total: 3790
 
+#define SAVE_TYPE_ERROR_SCREEN              FALSE   // When enabled, this shows an error message when the game is loaded on a cart without a flash chip or on an emulator with the wrong save type setting instead of crashing.
```

Comment:
I'm not sure if this is the best place to put this, as this file is relevant for changing the contents of the saveblock and nothing else. Maybe the config would be better in the general config file?

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5188/commits/ef5c47af5e9c2deeddee50650086f3f8076ae475